database/insertData/01_template_C_userRegistration 2.js

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
should the name of this file have a 2 at the end ?

Reply:
Nope. Good catch. 3079b6ef